pkg/validation/validation.go

```diff
@@ -158,17 +158,17 @@ func ValidateFullyQualifiedObjectID(i interface{}, _ string) (s []string, errors
 	v, _ := i.(string)
 	if strings.Contains(v, ".") { //nolint:gocritic // todo: please fix this
 		tagArray := strings.Split(v, ".")
-		if len(tagArray) != 3 {
-			errors = append(errors, fmt.Errorf("%v, is not a valid id. If using period delimiter, three parts must be specified <db_name>.<schema_name>.<object_name>", v))
+		if len(tagArray) > 3 {
```

Comment:
I am not sure why this was changed as a part of this PR. Can you elaborate?

Reply:
Overall it seems like the function could be replaced by the newly added IsValidIdentifier function. I didn't see this one when adding IsValidIdentifier, but this one is used in 3 places, so that shouldn't be a big change.

Reply:
removed the function entirely